fabischolasi/fast-food-fast-v1
# #160793971 api endpoints for orders

**What does this PR do?**
  - this pr contain resources to add, edit and delete orders

**How should this be manually tested?**
  - clone this repo, add postman extension, run the application server.
  - on postman, put the right URL, and test if the endpoints are working.
  - make sure you have Postgres installed

**Any background context you want to provide?**
  - use of flask_restful and Blueprints

**What are the relevant pivotal tracker stories?**
  #160793971
